Fix typo in README doc (#417)
10up/safe-redirect-manager: README.md

### Redirect loops detection

> By default redirect loop detection is enabled. To prevent redirect loops you can filter `srm_check_for_possible_redirect_loops`.

Reviewer: ```suggestion
By default redirect loop detection is enabled. To disable it, you can use the  `srm_check_for_possible_redirect_loops` filter.
```
